### PR TITLE
feat: Updated the settings screen to reflect the Figma design

### LINKED ISSUE #1272: feat(npt flutter): Settings screen relay atSign field validation

### PR DESCRIPTION
**Pull request by CurtlyCritchlow**
**- What I did**

- Implemented language support for English, Spanish and Portuguese
- Implemented theming based on Figma design
- implemented UI changes on settings Screen based on Figma design

**- How I did it**

**- How to verify it**
Run the app and change the language in the settings screen.
**- Description for the changelog**

### CODE REVIEW

**Review by xavierchanth — APPROVED** [review]

File: packages/dart/sshnp_flutter/lib/src/utility/constants.dart
Comment (xavierchanth) [inline packages/dart/sshnp_flutter/lib/src/utility/constants.dart:1]: Why was this file changed?

**Review by xavierchanth — CHANGES_REQUESTED** [review]
Please undo the changes outside of npt_flutter

File: packages/dart/sshnp_flutter/lib/src/utility/constants.dart
Comment (CurtlyCritchlow) [reply packages/dart/sshnp_flutter/lib/src/utility/constants.dart:1]: mistake. It was reverted. Thanks for the review

**Review by xavierchanth — APPROVED** [review]
